app/models/submission.rb

```diff
@@ -142,6 +142,15 @@ def cache_access_level
   scope :final_is_restricted_institution, -> { where('status LIKE "released for publication%"').where(access_level: 'restricted_to_institution') }
   scope :final_is_withheld, -> { where('status LIKE "released for publication%"').where(access_level: 'restricted') }
   scope :ok_to_release, -> { where('released_for_publication_at <= ?', Time.zone.today.end_of_day) }
+  scope :ok_to_autorelease, -> {
+                              ok_to_release.where('released_metadata_at <= ?', Time.zone.today.years_ago(2).end_of_day)
+                                           .where(access_level: 'restricted')
+                            }
+  scope :notify_author_of_upcoming_release, -> {
+                                              where('released_metadata_at >= ?', Time.zone.today.years_ago(2).end_of_day)
+                                                .where('released_for_publication_at <= ?', Time.zone.today.next_month)
+                                                .where(author_release_warning_sent_at: nil)
+                                            }
```

Comment:
I like how `ok_to_autorelease` is an adjective...very clear that these are the submissions that are free to go out into the world. Should we match it with the next one, something like `release_notification_needed`? Since it's a verb, it reads a little bit like calling this will notify the author, instead of just listing the submissions where we should notify the author.

Reply:
Good suggestion, I went with `release_warning_needed?` just so it doesn't get confused with the notification that goes out after a submission is released